ports/nrf/supervisor/qspi_flash.c

```diff
@@ -91,8 +89,7 @@ bool spi_flash_write_data(uint32_t address, uint8_t* data, uint32_t length) {
 }
 
 bool spi_flash_read_data(uint32_t address, uint8_t* data, uint32_t length) {
-    nrfx_qspi_read(data, length, address);
-    return true;
+    return nrfx_qspi_read(data, length, address) == NRFX_SUCCESS;
```

Comment:
maybe we should check for NRFX_RETURN for spi_flash_write_command and spi_flash_read_command as well

Reply:
Done.